src/command.js

```diff
@@ -1,11 +1,41 @@
 import path from 'path'
-import { parseOptions } from './options'
+import { parseOptions, optionDefaults } from './options'
+import { validateJSONPath } from './utils'
 
 const JQ_PATH = process.env.JQ_PATH || path.join(__dirname, '..', 'bin', 'jq')
 
+const getFileArray = path => {
+  if (Array.isArray(path)) {
+    return path.reduce((array, file) => {
+      validateJSONPath(file)
+      return [...array, file]
+    }, [])
+  }
+  validateJSONPath(path)
+  return [path]
+}
+
 export const commandFactory = (filter, json, options = {}) => {
+  const mergedOptions = {
+    ...optionDefaults,
+    ...options
+  }
+  let args = [filter, ...parseOptions(mergedOptions)]
+  let stdin = ''
```

Comment:
I'm concerned that not using null here and distinguishing it in exec() may cause a new regression(jq hanging) for attempts to do an actual input text of '' which is an often used pattern with jq.